### PR TITLE
More README about OCF-specific things

### LINKED ISSUE #9: write documentation for how to update DNS
running the makefile, why updating the serial matters, etc.

### PR DESCRIPTION
**Pull request by chriskuehl**
Fixes #9

### CODE REVIEW

**Comment by tahabi** [comment]
awesome

perhaps for documenting in docs/staff/backend/ldap/, how would we add new DNS records to LDAP? (obviously) I am not familiar with how this works.

File: README.md [hunk 1]
```diff
@@ -4,16 +4,62 @@
...
+
+
+## Making changes
+
+When updating DNS, apply your changes to LDAP, and then run `make`. It will
+rebuild the primary DNS (`db.ocf`) and reverse DNS, including pulling host data
+from LDAP and bumping the serial number.
+
+Sanity-check the diff before commiting to make sure the changes in LDAP make
+sense. If not, don't push!
```
Comment (matthew-mcallister) [inline README.md]: Ideally everyone would do this instinctively, even for automatically generated code, but, hey, it doesn't hurt to remind.

File: README.md [hunk 2]
```diff
@@ -4,16 +4,62 @@
...
 
+* `dnsA` for A and AAAA records.
 
-Make supports:
-- all
-- build
-- venv
-- test
-- install-hooks
+  Whether the host will automatically gets IPv6 records depends on the type of
```
Comment (chriskuehl) [inline README.md]: fixd

**Comment by chriskuehl** [comment]
There's documentation on adding new hosts which is probably helpful: https://www.ocf.berkeley.edu/docs/staff/procedures/new-host/

**Pull request merged**